### PR TITLE
Add kGWASflow v1.2.1

### PR DESCRIPTION
**Pull request by akcorut**
Describe your pull request here

This PR adds [kGWASflow](https://github.com/akcorut/kGWASflow) to bioconda. kGWASflow is a Snakemake workflow for performing k-mers-based GWAS. 

Kivanc Corut. akcorut/kGWASflow: v1.2.1. (2023). https://doi.org/10.5281/zenodo.7860705

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by kdm9** [comment]
@akcorut Neat pipelien! The build is failing as you have no setup.py/pyproject. Since you're packaging a snakemake workflow, you need to configure how the python wheel will be set up to contain all the snakemake stuff. One approach is to adapt what I have in https://github.com/kdm9/Acanthophis, whereby the python cli of the package copies the snakemake files to pwd from the python site-packages dir, then runs the workflow.

**Comment by akcorut** [comment]
Hi @kdm9, 

Thanks for the suggestions. I have added the setup.py and made the necessary changes. The new build seems to be working now. I hope it is good to go.

Thanks,
Kivanc

File: recipes/kgwasflow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - mamba
+    - numpy =1.23.5
+    - pandas =1.5.3
+    - snakemake =7.25.0
+    - click
+
+test:
+  commands:
+    - kgwasflow --help
+    - kgwasflow --version
```
Comment (kdm9) [inline recipes/kgwasflow/meta.yaml:34]: Please add ` - kgwasflow test ....` as appropriate to the tests that get run, if the test dataset is small enough to run in ~1 minute or so.

**Review by kdm9 — APPROVED** [review]

**Comment by kdm9** [comment]
@akcorut yep, looks good to me. I'll approve it, but one comment above: please add the `kgwasflow test` workflow to the list of conda tests, assuming it doesn't take many resources or time.

**Pull request merged**